apps/docs/docs/user/intro/_category_.json

```
@@ -0,0 +1,8 @@
+{
+  "label": "Introduction to Jayvee",
+  "position": 1,
+  "link": {
+    "type": "generated-index",
+    "description": "All the essentials Information to get started with Javee."
```

Please proofread these texts a bit ;).

```suggestion
    "description": "All the essential information to get started with Jayvee."
```